guides/v2.1/cloud/basic-information/starter-architecture.md

```diff
@@ -104,24 +110,26 @@ This software is *not* upgradable but versions for the following software is con
 * [RabbitMQ]({{ page.baseurl }}/cloud/project/project-conf-files_services-rabbit.html)
 * [Elasticsearch]({{ page.baseurl }}/cloud/project/project-conf-files_services-elastic.html)
 
-For Staging and Production, you will use Fastly for CDN and caching. We recommend installing Fastly module 1.2.33 or later. For details, see [Fastly in Cloud]({{ page.baseurl }}/cloud/basic-information/cloud-fastly.html).
+In the Staging and Production environments, you use Fastly for CDN and caching. We recommend installing Fastly module 1.2.33 or later. See [Fastly in Cloud]({{ page.baseurl }}/cloud/basic-information/cloud-fastly.html).
 
-For detailed information on supported versions and extensions, see the following information. These files allow you to configure software versions you want to use in your implementation.
+You use the following files to configure the software versions that you want to use in your implementation.
 
 *	[`.magento.app.yaml`]({{ page.baseurl }}/cloud/project/project-conf-files_magento-app.html)
 *	[`routes.yaml`]({{ page.baseurl }}/cloud/project/project-conf-files_routes.html)
 *	[`services.yaml`]({{ page.baseurl }}/cloud/project/project-conf-files_services.html)
 
 ### Backup and disaster recovery {#backup}
+
 You can create a snapshot of your database and file system using the Project Web Interface or the CLI. The snapshot includes your deployed code, installed software and services, and data. See [Snapshots and backup management]({{ page.baseurl }}/cloud/project/project-webint-snap.html).
 
 ## Prepare for development {#develop}
+
 To branch and develop your Magento store:
 
 * Set up your local environment
-* Clone the `master` branch from the Project to your local
+* Clone the `master` branch from the Project to your local environment
```

Comment:
The word environment is overused throughout, so we adopted “local workstation” to reduce confusion.